### PR TITLE
ADS DaxTextView: Support for android:textColor

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL: https://app.asana.com/0/1202857801505092/1203418290512953

### Description

- Support the textColor attribute from Android
- When textType and textColor are both set, textType will take priority.

### CODE REVIEW

File: common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/text/DaxTextView.kt [hunk 1]
```diff
@@ -40,33 +40,46 @@ constructor(
                 0,
             )
 
-        val typographyType = if (typedArray.hasValue(R.styleable.DaxTextView_typography)) {
-            Type.from(typedArray.getInt(R.styleable.DaxTextView_typography, 0))
+        val typographyTypography = if (typedArray.hasValue(R.styleable.DaxTextView_typography)) {
```
Comment (nalcalag) [inline common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/text/DaxTextView.kt:43]: I think there is a typo in the variable name

File: common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/text/DaxTextView.kt [hunk 2]
```diff
@@ -114,22 +127,22 @@ constructor(
         }
     }
 
-    enum class TextColor {
+    enum class Type {
```
Comment (nalcalag) [inline common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/text/DaxTextView.kt:130]: `TextType`?

**Comment by malmstein** [comment]
@nalcalag thanks for the review, I've added a missing lint rule too

**Review by nalcalag — APPROVED** [review]
Works as expected 💯

**Pull request merged**